internal/fs/inode/file.go

```diff
@@ -517,8 +517,24 @@ func (f *FileInode) Sync(ctx context.Context) (err error) {
 		return
 	}
 
+	// When listObjects call is made, we fetch data with projection set as noAcl
+	// which means acls and owner properties are not returned. So the f.src object
+	// here will not have acl information even though there are acls present on
+	// the gcsObject.
+	// Hence, we are making an explicit gcs stat call to fetch the latest
```

Comment:
Can update this comment to call out that Stat call sets the full projection.

Reply:
done